### ISSUE TITLE
single channel output

### ISSUE THREAD

**Issue by SreenijaK** [wkentaro/labelme#612]
How do i convert the labeled output from labelme to single channel output. will converting them to grayscale using python packages do the work for me?

**Comment by wkentaro** [comment]
I don't really understand 

- what is "labeled output" -- is it json file?
- single channel output -- is it gray scale PNG image?
- will converting them to grayscale using python packages do the work for me -- can you rephrase this? I don't catch that.
